src/tables/opentype.js

```diff
@@ -33,8 +33,13 @@ export let ScriptList = new r.Array(ScriptRecord, r.uint16);
 // Features and Lookups #
 //#######################
 
+let FeatureParams = new r.Struct({
+  version:    r.uint16, // should be set to 0 according OT spec
+  nameID:     r.uint16, //OT spec: UI Name ID or uiLabelNameId
+});
+
 export let Feature = new r.Struct({
-  featureParams:      r.uint16, // pointer
+  featureParams:      0 === r.uint16 ? 0 : new r.Pointer(r.uint16, FeatureParams),
```

Comment:
I don't think the comparison will work here. `r.uint16` is a type, not a value. `r.Pointer` will automatically handle the null pointer case.

```suggestion
  featureParams:      new r.Pointer(r.uint16, FeatureParams),
```